core/src/models/App.ts

```diff
@@ -269,6 +271,12 @@ export class App extends LoggedModel<App> {
       name,
       type,
       options,
+      refresh: appRefreshQuery
+        ? {
+            query: appRefreshQuery.refreshQuery,
+            recurringFrequency: appRefreshQuery.recurringFrequency,
```

Comment:
Is there more on `appRefreshQuery` than these two fields? Just curious.

Reply:
On the model, yes.  On the configObject (what this is building), no.

We do some similar things for schedules and sources [here](https://github.com/grouparoo/grouparoo/blob/0fb5e1761f691671f8d70fe0e8b4698b2de0e15c/core/src/models/Source.ts#L350).

Reply:
Could just do `refresh: appRefreshQuery` then. If it's not there it should be undefined. If it returns null, can do: `refresh: appRefreshQuery ?? null`

Reply:
Ah, `appRefreshQuery` has all of the fields from the model here (appId, timestamps, value, etc.), that's why I spelled out the two that should be in the config object.

Reply:
Ah, alright! All good then :)